src/neo/Network/P2P/MessageCommand.cs

```diff
@@ -20,7 +20,7 @@ public enum MessageCommand : byte
         Pong = 0x19,
 
         //synchronization
-        [ReflectionCache(typeof(GetBlockByIndexPayload))]
+        [ReflectionCache(typeof(GetBlocksPayload))]
```

Comment:
We can use index to sync headers too, right?

Reply:
Done